### PR TITLE
[doc] [dune] Some tweaks from #13617

### PR DESCRIPTION
**Pull request by ejgallego**
Tweaks to docs that are independent / unrelated to that PR.

### CODE REVIEW

File: Makefile.dune [hunk 1]
```diff
@@ -18,7 +18,7 @@ help:
 	@echo "  - world:  build all public binaries and libraries"
 	@echo "  - watch:  build all public binaries and libraries [continuous build]"
 	@echo "  - check:  build all ML files as fast as possible"
-	@echo "  - test-suite: run Coq's test suite"
+	@echo "  - test-suite: run Coq's test suite [use NJBOS=N envar to set job parallelism]"
```
Comment (SkySkimmer) [inline Makefile.dune:21]:
```suggestion
	@echo "  - test-suite: run Coq's test suite [env NJOBS=N to set job parallelism]"
```

**Review by SkySkimmer — APPROVED** [review]

File: Makefile.dune (hunk 1 above)
Comment (SkySkimmer) [reply Makefile.dune:21]: ping typo

Comment (ejgallego) [reply Makefile.dune]: Merci !

File: Makefile.dune [hunk 2]
```diff
@@ -18,7 +18,7 @@ help:
 	@echo "  - world:  build all public binaries and libraries"
 	@echo "  - watch:  build all public binaries and libraries [continuous build]"
 	@echo "  - check:  build all ML files as fast as possible"
-	@echo "  - test-suite: run Coq's test suite"
+	@echo "  - test-suite: run Coq's test suite [env NJBOS=N to set job parallelism]"
```
Comment (SkySkimmer) [inline Makefile.dune:21]:
```suggestion
	@echo "  - test-suite: run Coq's test suite [env NJOBS=N to set job parallelism]"
```

Comment (ejgallego) [reply Makefile.dune:21]: O_O :abacus: :cactus: :dancing_men:

**Comment by SkySkimmer** [comment]
@coqbot: merge now

**Pull request merged**